fdbserver/worker.actor.cpp

```diff
@@ -1261,6 +1261,18 @@ ACTOR Future<Void> workerServer(Reference<ClusterConnectionFile> connFile,
 					flushAndExit(0);
 				}
 			}
+			when(SetFailureInjection req = waitNext(interf.clientInterface.setFailureInjection.getFuture())) {
+				if (!FLOW_KNOBS->ENABLE_CHAOS_FEATURES) {
+					req.reply.sendError(client_invalid_operation());
```

Comment:
Is it possible to still run the conditionals below even if `ENABLE_CHAOS_FEATURES` is disabled?

Reply:
you're right, this is a bug! Thanks for catching this. I fixed it